ow_lander/src/ow_lander/server.py

```diff
@@ -80,7 +80,7 @@ def _set_succeeded(self, msg, **kwargs):
     kwargs -- Keyword arguments that match fields in result_type. Leave empty
               if result type is empty.
     """
-    rospy.loginfo(self.__format_result_msg(f"{self.name}: Succeded", msg))
+    rospy.loginfo(self.__format_result_msg(f"{self.name}: Succeeded", msg))
```

Comment:
Thanks! Good catch. Amazing how many instances of this message I have looked at now and did not notice this